modules/rpc/yarpc.go

```diff
@@ -121,7 +121,6 @@ func (m *YarpcModule) Start(readyCh chan<- struct{}) <-chan error {
 
 	interceptor := yarpc.Interceptors(m.interceptors...)
 
-	// TODO(ai/madhu) pass option for opentracing to NewDispatcher
 	m.rpc, err = _dispatcherFn(m.Host(), yarpc.Config{
```

Comment:
I meant to comment on the other diff. Since we're embedding `ModuleBase` directly (e.g. composing it), we should access the tracer directly instead of calling `.ModuleBase.Tracer()`

e.g..

```go
Tracer: m.Tracer(),
```

It's slightly cleaner and makes it easier to refactor